Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,63 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+# Tips and Tricks
+
+This is a list of useful tips and tricks to help you make the most out of SerenityOS
+
+Related: [Keyboard Shortcuts](help://man/7/KeyboardShortcuts)
+
+## General
+* When on the Desktop or in File Manager, start typing and the nearest match will be selected
+* Bold text in context menus hints at the default behavior of a double-click
+* Hold `Ctrl` to accelerate mouse wheel interaction with sliders and spin boxes
+* Hold `Ctrl` while activating a menu item to prevent that menu from closing
+* Press `Ctrl+Shift+A` on a focused widget or application to show the Command Palette, a searchable list of available actions
```

Comment:
That's a general application shortcut, and an important one! It should be part of KeyboardShortcuts instead.